src/host/tcp.h

```diff
@@ -161,6 +175,29 @@ namespace asynchost
       return service;
     }
 
+    void client_bind()
```

Comment:
Some minor refactor of the client bind as we didn't re-bind to the client interface in the disconnection case.